api/openapi-spec/swagger.json

```diff
@@ -5540,10 +5540,21 @@
    },
    "v1.VirtualMachineInstanceNetworkInterface": {
     "properties": {
+     "interfaceName": {
+      "description": "The interface name on the Virtual Machine",
```

Comment:
I suppose "inside the Virtual Machine" or which name is this about?

Reply:
Done